### PR TITLE
creates level select dropdown without any functionality

### LINKED ISSUE #7: Dummy Sidebar Level Select
Create level select component however with no functionality. You can implement this into the sidebar component however do not add any logic to make sure no merge conflicts occur. Maybe try using react-dropdown (already installed) for dropdown part.

### PR DESCRIPTION
**Pull request by ldalton02**
Creates a level select dropdown currently without any function. Not exactly how I want it to look when finished, I'm thinking about changing up the arrow icon or some of the style. Also, did not yet add the side arrows to switch to the next or previous level that are in the figma.

The changes to package.json and yarn.lock are because I installed the react-icons package to put an arrow icon in the dropdown

closes #11

### CODE REVIEW

**Review by nendow02 — APPROVED** [review]
This is looking really good!

**Comment by nendow02** [comment]
Go ahead and squash and merge, if there’s a merge conflict lmk. Good work!

File: src/components/levelSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import {useState} from 'react';
+import Dropdown, { Option } from 'react-dropdown';
+import { BsArrowUpShort, BsArrowDownShort } from 'react-icons/bs';
+
+
+function LevelSelect(): JSX.Element {
+
+  const pages = ['1', '2', '3', '4', '5', '6', '7', '8', '9'];
+
+  const [currentPage, setCurrentPage] = useState(6 /* change this to current page once set up */);
```
Comment (reginawang3495) [inline src/components/levelSelect.tsx]: nit: generally, we probably wouldn't like putting comments inside of the fuction!
```suggestion
  const [currentPage, setCurrentPage] = useState(6); // change this to current page once set up
```

File: src/components/levelSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  const DropdownArrowDown = <BsArrowDownShort/>;
+
+  const createDropDownItems = ():Option[] => {
+    const myArray = pages.map((item) => {
+      return  ({
+        label: <div>{item}</div>,
+        value: item,
+      });
+    });
+    return myArray;
```
Comment (reginawang3495) [inline src/components/levelSelect.tsx]: nit:
```suggestion
    return pages.map((item) => {
      return  ({
        label: <div>{item}</div>,
        value: item,
      });
    });
```

File: src/components/levelSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+
+function LevelSelect(): JSX.Element {
+
+  const pages = ['1', '2', '3', '4', '5', '6', '7', '8', '9'];
+
+  const [currentPage, setCurrentPage] = useState(6 /* change this to current page once set up */);
+
+  const DropdownArrowUp = <BsArrowUpShort/>;
+  const DropdownArrowDown = <BsArrowDownShort/>;
```
Comment (reginawang3495) [inline src/components/levelSelect.tsx]: nit: there's only one use case of this, why are we taking it out of the other area?

**Review by reginawang3495 — COMMENTED** [review]
looks really good!! this is just totally unimportant small code cleanup ideas ;) (it'll be useful in future esp since when the codebase gets bigger + also future LLs)

**Comment by reginawang3495** [comment]
also, you might want to add a "closes #\<your ticket\>" in your description btw!

**Pull request merged**